source/Generic/PlayState/ViewModels/PlayStateManagerViewModel.cs

```diff
@@ -415,12 +423,21 @@ public bool SwitchGameState(PlayStateData gameData)
                         notificationType = processesSuspended ? NotificationTypes.Resumed : NotificationTypes.PlaytimeResumed;
                         gameData.Stopwatch.Stop();
                         logger.Debug($"Game {gameData.Game.Name} resumed in mode {gameData.SuspendMode}");
+                        if (settings.Settings.NotificationShowSuspendedReminder)
+                        {
+                            gameData.SetReminderTimer();
+                        }
                     }
                     else
                     {
                         gameData.IsSuspended = true;
                         notificationType = processesSuspended ? NotificationTypes.Suspended : NotificationTypes.PlaytimeSuspended;
                         gameData.Stopwatch.Start();
+                        if (settings.Settings.NotificationShowSuspendedReminder)
```

Comment:
I think the functionality should be able to be enable for both suspended and not suspended games, in different options for more flexibility and not only for suspended games

Reply:
I'm also not sure that the subscribing and everything needed should be in this method, why is it done here?

Reply:
You mean a reminder that reminds you that you been playing a game for x minutes, right?
If so, should the reminder time be the same for both scenarios, or be configurable for each case? (example, 2 minutes reminder for suspended game, 30 minutes for played game).

Reply:
> I'm also not sure that the subscribing and everything needed should be in this method, why is it done here?

The idea is that when you pause the game, after for example 5 minutes you'll get the notification. If you resume the game, after the second pause the reminder should start from 0. If there is another suspension trigger I could move the method there  but since the Soptwatch was started there too for me was logical to add it there too.